runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -2465,6 +2465,23 @@ private static String longBossName(String boss)
 			case "pm":
 				return "Phantom Muspah";
 
+			// Desert Treasure 2 bosses
+			case "the leviathan":
```

Comment:
This is covered by the default case; `WordUtils.capitalize()` would transform "the leviathan" to "The Leviathan" already. The same is true for the other entries. The case matching here is only for shortnames/nicknames, so "leviathan", "duke", "sucellus", and "whisperer" are all good additions.